UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs

```diff
@@ -45,7 +46,7 @@ void Start()
 
 	public void RunOreGenerator()
 	{
-		var metaTileMap = GetComponentInChildren<MetaTileMap>();
+		metaTileMap = GetComponentInChildren<MetaTileMap>();
 		wallTilemap = metaTileMap.Layers[LayerType.Walls].GetComponent<Tilemap>();
```

Comment:
BoundsInt bounds = wallTilemap.cellBounds;
can you add TODO for moving this to metaTileMap